csharp/src/Ice/UdpEndpoint.cs

```diff
@@ -44,7 +43,7 @@ public override bool Equals(Endpoint? other)
                 return true;
             }
             return other is UdpEndpoint udpEndpoint &&
-                HasCompressionFlag == udpEndpoint.HasCompressionFlag &&
+                _hasCompressionFlag == udpEndpoint._hasCompressionFlag &&
```

Comment:
I think keeping `_hasCompressionFlag` in Equals and GetHashCode was an oversight in Benoit's PR.

Reply:
Fixed